components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2366,6 +2367,9 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             return costCenter?.spendingLimit;
         } catch (error) {
             log.error(`Failed to subscribe '${attributionId}' to Stripe`, error);
+            if (error instanceof ClientError) {
+                throw new ResponseError(error.code, error.details);
```

Comment:
This looks like we're starting to spill system internals to clients (again). What's the reason for doing so? Only for debuggin/testing? :thinking:

Reply:
It's a user error to provide a non-working credit card the error message from stripe is telling the user what happened i.e. why their credit card doesn't work.